src/Psalm/Storage/ClassLikeStorage.php

```diff
@@ -373,7 +373,7 @@ class ClassLikeStorage
     public $initialized_properties = [];
 
     /**
-     * @var array<string>
+     * @var array<string, true>
```

Comment:
This is BC break. Good thing that the PR already targets master.

Reply:
Oh, good point, I'd completely forgotten about plugins. I'll take more care about that in the future.

Reply:
I don't think that change is really needed anymore, but unless you object I don't really see a need to change it back. It seems better to me for it to be a set rather than a list, and `array<string, true>` seems like the closest thing PHP has to a native string set without using the ds extension.
Or maybe it'd be better to change it back and methodically update a lot of these types of properties to be more set-like in a separate pull request instead?